src/Bicep.Local.Deploy/LocalDeploymentEngineHost.cs

```diff
@@ -48,6 +50,12 @@ public class LocalDeploymentEngineHost : DeploymentEngineHostBase
 {
     private readonly LocalExtensibilityHandler extensibilityHandler;
 
+    /*var extensionName = requestUri.Segments[^4].TrimEnd('/');
+    var extensionVersion = requestUri.Segments[^3].TrimEnd('/');
+    var method = requestUri.Segments[^1].TrimEnd('/');*/
```

Comment:
[nit] Remove this

Reply:
(facepalm) moved to copy the arg names and omit to remove these.